apps/website/src/content/docs/code.mdx

```diff
@@ -1,18 +1,17 @@
 ---
 title: Code
-description:
-propsPath: '@itwin/itwinui-react/esm/core/Typography/Code.d.ts'
+description: Showcase code snippets in line with the body text.
```

Comment:
I think this also need to be updated according to https://github.com/iTwin/iTwinUI/pull/1766/files#r1441867734

Reply:
Can be rephrased to be more concise?

```suggestion
description: Display a short piece of code within some existing text.
```